test/integration/clusterio.js

```diff
@@ -27,6 +27,34 @@ async function checkInstanceStatus(id, status) {
 	assert.equal(instances.get(44).status, status, "incorrect instance status");
 }
 
+async function startAltHost() {
+	let config = "alt-host-config.json";
+	let configPath = path.join("temp", "test", config);
+	await fs.remove(configPath);
+	await fs.remove(path.join("temp", "test", "alt-instances"));
+	await execCtl(`host create-config --id 5 --name alt-host --generate-token --output ${config}`);
```

Comment:
Any reason why `5` was selected as the id? Would it be worth having it as an argument to this function?

Reply:
It used to be 5 before the code was moved to this function. 5 was selected because it needed a number and 5 sounded nice.